Sources/FluentUI_iOS/Components/Command Bar/CommandBar.swift

```diff
@@ -58,15 +58,15 @@ public class CommandBar: UIView, TokenizedControl {
                 return nil
             }
 
-            return [[leadingItem]]
+            return [CommandBarItemGroup(items: [leadingItem])]
```

Comment:
TIL about [ExpressibleByArrayLiteral](https://developer.apple.com/documentation/swift/expressiblebyarrayliteral), which I *think* would mean we could make this change without breaking most uses of the API?

Reply:
unfortunately only works with swift usages but still cleans up the changes we need in our demo controllers!